python/cuml/dask/linear_model/logistic_regression.py

```diff
@@ -31,7 +32,8 @@
 np = cpu_only_import("numpy")
 
 
-class LogisticRegression(BaseEstimator, SyncFitMixinLinearModel):
+# class LogisticRegression(BaseEstimator, SyncFitMixinLinearModel):
+class LogisticRegression(LinearRegression):
     def __init__(self, *, client=None, verbose=False, **kwargs):
```

Comment:
I do think there's a case to be made for specifying defaults in the constructor at this layer, and they should end up getting propagated to the cython class. Another thing we missed in the review of the initial Dask estimator is that this should be documented at the class level. Can you add those docs in this PR too? I think most of the docs can likely be copied from the single gpu `LogisticRegression`

Reply:
Good suggestion. I have revised the __init__ function and added class-level documentation.